scripts/tw-make

```diff
@@ -109,9 +138,31 @@ if __name__ == "__main__":
     if args.subcommand == "custom":
         options.nb_rooms = args.world_size
         options.nb_objects = args.nb_objects
-        options.quest_length = args.quest_length
-        options.quest_breadth = args.quest_breadth
-        game_file, game = textworld.make(options)
+        options.nb_parallel_quests = args.nb_parallel_quests
+        options.chaining.min_breadth = args.quest_min_breadth
+        options.chaining.min_depth = args.quest_min_depth
+        options.chaining.min_length = args.quest_min_length
+        options.chaining.max_breadth = args.quest_max_breadth
+        options.chaining.max_depth = args.quest_max_depth
+        options.chaining.max_length = args.quest_max_length
+
+        if args.quest_length:
+            options.chaining.min_length = args.quest_length
+            options.chaining.max_length = args.quest_length
+            options.chaining.max_depth = args.quest_length
+
+        if args.quest_breadth:
+            options.chaining.min_breadth = args.quest_breadth
+            options.chaining.max_breadth = args.quest_breadth
+
+        try:
+            game_file, game = textworld.make(options)
+        except TextworldNoSuchQuestExistError:
+            # TODO: Display current values of `options` to help understanding why no game could be generated.
```

Comment:
@tavianator if you have any idea of how to properly display the GameOptions/ChainingOptions/GrammarOptions, I'm all ears.